lib/synced/timestamp.rb

```diff
@@ -0,0 +1,9 @@
+class Synced::Timestamp < ActiveRecord::Base
+  self.table_name = 'synced_timestamps'
+  belongs_to :parent_scope, polymorphic: true
+  scope :with_scope, ->(parent_scope) { where(parent_scope: parent_scope) }
+
+  def self.last_synced_at
+    maximum(:synced_at)
```

Comment:
why would you order by id? we need latest synced_at, not latest id, even if the order seems similar so far :trollface: